build.sbt

```diff
@@ -1,12 +1,31 @@
-name := "openie"
-
-organization := "edu.washington.cs.knowitall.openie"
-
-crossScalaVersions := Seq("2.11.8")
-
-scalaVersion <<= crossScalaVersions { (vs: Seq[String]) => vs.head }
+import org.allenai.plugins.CoreDependencies
+
+lazy val buildSettings = Seq(
+  organization := "org.allenai.openie",
+  crossScalaVersions := Seq(CoreDependencies.defaultScalaVersion),
+  scalaVersion <<= crossScalaVersions { (vs: Seq[String]) => vs.head },
+  publishMavenStyle := true,
+  publishArtifact in Test := false,
+  pomIncludeRepository := { _ => false },
```

Comment:
I was mostly copying from https://github.com/allenai/common/blob/master/build.sbt ... 

Removing `pomIncludeRepository := { _ => false },` causes this to appear in `openie_2.11.pom`:

```
    <repositories>
        <repository>
            <id>bintrayallenaimaven</id>
            <name>bintray-allenai-maven</name>
            <url>https://dl.bintray.com/allenai/maven/</url>
            <layout>default</layout>
        </repository>
        <repository>
            <id>spray</id>
            <name>spray</name>
            <url>http://repo.spray.io/</url>
            <layout>default</layout>
        </repository>
        <repository>
            <id>TypesafeReleases</id>
            <name>Typesafe Releases</name>
            <url>http://repo.typesafe.com/typesafe/releases/</url>
            <layout>default</layout>
        </repository>
        <repository>
            <id>AI2PublicReleases</id>
            <name>AI2 Public Releases</name>
            <url>http://utility.allenai.org:8081/nexus/content/repositories/public-releases/</url>
            <layout>default</layout>
        </repository>
    </repositories>
```

Looks like we have this set in a number of repositories:

https://github.com/search?q=org%3Aallenai+pomIncludeRepository&type=Code

Does that change your mind or would you still like this directive removed (and these POM entries added)?